hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/TypedTable.java

```diff
@@ -456,6 +456,27 @@ public long getEstimatedKeyCount() throws IOException {
     return rawTable.getEstimatedKeyCount();
   }
 
+  /**
+   * Returns the exact key count of column family.
+   *
+   * @return keyCount
+   * @throws IOException
+   */
```

Comment:
No need to duplicate doc in implementation.

```suggestion
```

Reply:
Handled the review comments. Kindly re-review.